### PR TITLE
Upgrade Vector

### PR DESCRIPTION
**Pull request by razvan**
CI: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/trino-operator-it-custom/88/

# Description

*Please add a description here. This will become the commit message of the merge request later.*

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [x] CRD changes approved
- [x] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [x] Changelog updated
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: tests/templates/kuttl/logging/trino-vector-aggregator-values.yaml.j2 [hunk 1]
```diff
@@ -104,13 +104,23 @@ customConfig:
...
+      inputs: [vector]
+      condition: |-
+        .timestamp == from_unix_timestamp!(0) ||
+        is_null(.level) ||
+        is_null(.logger) ||
+        is_null(.message)
   sinks:
     out:
-      inputs: [automaticLogConfig*, customLogConfig*]
+      inputs: [filtered*, custom*]
```
Comment (siegfriedweber) [inline tests/templates/kuttl/logging/trino-vector-aggregator-values.yaml.j2:117]: Why didn't you also prefix the custom ones with "filtered"?

Comment (razvan) [reply tests/templates/kuttl/logging/trino-vector-aggregator-values.yaml.j2:117]: To preserve the previous logical separation between `custom` and `automatic`. I didn't look at the details of the `custom` entries. Should I do it ?

Comment (siegfriedweber) [reply tests/templates/kuttl/logging/trino-vector-aggregator-values.yaml.j2:117]: There is no need to look at the details. Just prefix all rules with `filtered` (see e.g. https://github.com/stackabletech/druid-operator/pull/459/files#diff-d35089690058957988f8dd6b3bcf295ab343dd5fe8db9381dbd1daeb3c537728).

Comment (razvan) [reply tests/templates/kuttl/logging/trino-vector-aggregator-values.yaml.j2]: Done https://github.com/stackabletech/trino-operator/pull/453/commits/fcdae35357cab1995e108259dea47f72cc754c4c

**Review by siegfriedweber — APPROVED** [review]

**Pull request merged**